src/skills-builder/skills-builder-header/messages.js

```diff
@@ -11,6 +11,11 @@ const messages = defineMessages({
     defaultMessage: 'Let edX be your guide',
     description: 'Subheading to the Skills Builder title in the header component',
   },
+  skillsBuilderHeaderTitleIsMedium: {
+    id: 'skills.builder.header.title.is.Medium',
```

Comment:
nit: should `Medium` be lowercase?

Reply:
Good catch on this, too! Didn't intend for that to be uppercase